OracleCloudInfrastructure/oci-cli/README.md

```diff
@@ -93,6 +93,13 @@ $ docker run --rm -it -v "$HOME/.oci:/oracle/.oci" oci --auth instance_principal
 
  If you created a shell alias, add it to the alias definition.
 
+## Access local files from OCI container
+For OCI container to access local files, create a "scratch" directory in your home directory and map that into the container, for example this command does bulk upload of local fiels to specified OCI bucket:
```

Comment:
The "for example" should be a new sentence.

fiels -> files

specified -> the specified

Reply:
Thanks Ryan for your review. 
The suggested comments have been incorporated.